src/PDB/PDBParser.jl

```diff
@@ -150,7 +150,7 @@ const _Format_PDB_ATOM = FormatExpr(
     #          1         2         3         4         5         6         7         8
     # 12345678901234567890123456789012345678901234567890123456789012345678901234567890
     # >    <>   < >  <|> < |>  <|   >      <>      <>      <>    <>    <      >  <><><
-    "{:<6}{:>5d} {:<4}{:>1}{:>3} {:>1}{:>4}{:>1}   {:>8.3f}{:>8.3f}{:>8.3f}{:>6.2f}{:>6.2f}      {:<4}{:>2}{:>2}\n"
+    "{:<6}{:>5d} {:<4}{:>1}{:>3} {:>1}{:>4}{:>1}   {:>8.3f}{:>8.3f}{:>8.3f}{:>6.2f}{:>6}      {:<4}{:>2}{:>2}\n"
```

Comment:
Hi @timholy , 

Is there a reason to have more than 2 decimal places for the B-factors? 

The [PDB format description](https://www.wwpdb.org/documentation/file-format-content/format23/sect9.html) describe this field as:
```
COLUMNS      DATA TYPE        FIELD      DEFINITION
------------------------------------------------------
61 - 66      Real(6.2)        tempFactor Temperature factor.
```

Reply:
Oh shoot, I don't know why I did that.